### PR TITLE
feat: change configs and add commercetools sdk

### PR DESCRIPTION
**Pull request by leonmargaritis**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -5,3 +5,10 @@ SITE_NAME="Next.js Commerce"
 SHOPIFY_REVALIDATION_SECRET=""
```
Comment (KT-Anja) [inline .env.example:5]: Will the SHOPIFY envs be needed?

Comment (leonmargaritis) [reply .env.example:5]: We could run shopify and commercetools in parallel and then swap the functions one after the other - until shopify is no longer necessary. What do you think?

Comment (KT-Anja) [reply .env.example:5]: haha, yes
I simply forgot the intended use

File: pnpm-lock.yaml
Comment (KT-Anja) [inline pnpm-lock.yaml:1]: Depending on which package manager the team chooses, this file needs to be exchanged.

**Review by KT-Anja — APPROVED** [review]

**Pull request merged**